orcid_hub/models.py

```diff
@@ -27,13 +27,12 @@
 from pycountry import countries
 from pykwalify.core import Core
 from pykwalify.errors import SchemaError
-
 from peewee_validates import ModelValidator
 
 from . import app, db
 from .config import DEFAULT_COUNTRY, ENV
 
-EMAIL_REGEX = re.compile(r"^[_a-z0-9-]+(\.[_a-z0-9-]+)*@[a-z0-9-]+(\.[a-z0-9-]+)*(\.[a-z]{2,4})$")
+EMAIL_REGEX = re.compile(r"^[_a-z0-9-+]+(\.[_a-z0-9-]+)*@[a-z0-9-+]+(\.[a-z0-9-]+)*(\.[a-z]{2,4})$")
```

Comment:
it looks that this REGEX needs to be improved to include all valid cases: https://en.wikipedia.org/wiki/Email_address#Local-part and also need to take a look at the rest of the RFCs (https://en.wikipedia.org/wiki/Email_address#Local-part)